implementations/rust/ockam/ockam_app/src/shared_service/tcp_outlet/create.rs

```diff
@@ -30,15 +33,19 @@ pub async fn tcp_outlet_create(
 async fn tcp_outlet_create_impl(
     app: AppHandle<Wry>,
     service: String,
-    port: String,
+    address: String,
     email: Option<String>,
 ) -> crate::Result<()> {
-    debug!(%service, %port, "Creating an outlet");
+    debug!(%service, %address, "Creating an outlet");
     let app_state = app.state::<AppState>();
-    let socket_addr: SocketAddr = format!("127.0.0.1:{port}")
-        .parse()
+    let addr = if let Some((host, port)) = address.split_once(':') {
+        format!("{host}:{port}")
+    } else {
+        format!("{DEFAULT_HOST}:{address}")
+    };
+    let socket_addr = resolve_peer(addr)
         .into_diagnostic()
-        .wrap_err("Invalid port")?;
+        .wrap_err("Invalid address. The expected formats are 'host:port' and 'port")?;
```

Comment:
After re-reading your commit description, I realized we were missing a format here. Let's change this error message to `Invalid address. The supported formats are 'host:port', 'ip:port' and 'port'`.